### PR TITLE
Rita's review of chapter 8

### PR DESCRIPTION
**Pull request by ritaf-ORM**

### CODE REVIEW

File: chapter_08_prettification.asciidoc [hunk 1]
```diff
@@ -804,19 +808,21 @@ I think that looks great!
 .The lists page goes dark
 image::images/prettified-dark.png["Screenshot of lists page in dark mode. Cool."]
 
+//RITA: Whew. I was more concerned about the legibility of dark mode figures when printed, but thanks all the same.
 But it's very much a matter of personal preference,
 and my editor will kill me
```
Comment (ritaf-ORM) [inline chapter_08_prettification.asciidoc:813]: On second thought, would you consider changing this to "my editor will be mad at me" or even some other silly description (freak out, self-combust, explode, etc)? I'd rather not imply that we'll cause you harm.

Comment (hjwp) [reply chapter_08_prettification.asciidoc:813]: haha i don't think anyone would take it seriously, but sure i don't mind :)

File: chapter_08_prettification.asciidoc [hunk 2]
```diff
@@ -529,7 +533,7 @@ of the grid system and the `justify-content-center` class to get what we want:
 (If you've never seen an HTML tag broken up over several lines,
 that `<input>` may be a little shocking.
 It is definitely valid,
-but you don't have to use it if you find it offensive. ;)
+but you don't have to use it if you find it offensive.)
```
Comment (hjwp) [inline chapter_08_prettification.asciidoc:536]:
```suggestion
but you don't have to use it if you find it offensive. ;)
```

File: chapter_08_prettification.asciidoc [hunk 3]
```diff
@@ -804,19 +808,21 @@ I think that looks great!
 .The lists page goes dark
 image::images/prettified-dark.png["Screenshot of lists page in dark mode. Cool."]
 
+//RITA: Whew. I was more concerned about the legibility of dark mode figures when printed, but thanks all the same.
```
Comment (hjwp) [inline chapter_08_prettification.asciidoc:811]:
```suggestion
//RITA: Whew. I was more concerned about the legibility of dark mode figures when printed, but thanks all the same.
//RITA: On second thought, would you consider changing this to "my editor will be mad at me" or even some other silly description (freak out, self-combust, explode, etc)? I'd rather not imply that we'll cause you harm.

```

File: chapter_08_prettification.asciidoc [hunk 4]
```diff
@@ -804,19 +808,23 @@ I think that looks great!
...
+//RITA: On second thought, would you consider changing this to "my editor will be mad at me" or even some other silly description (freak out, self-combust, explode, etc)? I'd rather not imply that we'll cause you harm.
+
 But it's very much a matter of personal preference,
 and my editor will kill me
 if I make all the rest of my screenshots use so much ink,
 so I'm going to revert it for now.
-You feel free to keep it if you like!
+You're free to keep dark mode on if you like!
```
Comment (hjwp) [inline chapter_08_prettification.asciidoc:818]: why this change?

File: chapter_08_prettification.asciidoc [hunk 5]
```diff
@@ -911,7 +919,7 @@ STATIC_ROOT = BASE_DIR / "static"
 Take a look at the top of the settings file,
 and you'll see how that `BASE_DIR` variable is helpfully defined for us,
 using `pathlib.Path` and `__file__`
-(both really nice Python builtins)footnote:[
+(both really nice Python built-ins)footnote:[
```
Comment (hjwp) [inline chapter_08_prettification.asciidoc:922]:
```suggestion
(both really nice Python builtins)footnote:[
```

File: chapter_08_prettification.asciidoc (hunk 4 above)
Comment (ritaf-ORM) [reply chapter_08_prettification.asciidoc:818]: Edit was to clarify what "it" referred to.

Comment (hjwp) [reply chapter_08_prettification.asciidoc:818]: aha makes sense.  thanks!
